plugins/Marketplace/resources/qml/PackagePage.qml

```diff
@@ -40,7 +40,7 @@ Rectangle
                     id: downloadCount
                     Layout.preferredWidth: parent.width
                     Layout.fillHeight: true
-
+                    visible: packageData.downloadCount != "0"
```

Comment:
Is this the only variable we can use to detect if we are on a download page? 

A plugin with 0 downloads, should still display 0 download. Now it doesn't.

Reply:
It was the only way i could come up with. I figured that the single user that would not see the download button was an acceptable issue.

Reply:
Maybe we can add a comment explaining why we did this?

Reply:
Yup! Done